packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -16,20 +16,32 @@ import {
     FieldLocation, TreeLocation,
 } from "../../forest";
 import { StoredSchemaRepository } from "../../schema";
-import { FieldKey, TreeType, DetachedRange } from "../../tree";
+import { FieldKey, TreeType, DetachedRange, AnchorSet } from "../../tree";
 import { brand } from "../../util";
 
 export class ObjectForest extends SimpleDependee implements IEditableForest {
     private readonly dependent = new SimpleObservingDependent(() => this.invalidateDependents());
-    public readonly anchors: Set<ObjectAnchor> = new Set();
+
+    public readonly schema: StoredSchemaRepository = new StoredSchemaRepository();
+    public readonly anchors: AnchorSet = new AnchorSet();
+
     public readonly root: Anchor = new RootAnchor();
     public readonly rootField: DetachedRange = this.newRange();
-    public readonly schema: StoredSchemaRepository = new StoredSchemaRepository();
 
     private readonly roots: Map<DetachedRange, ObjectField> = new Map();
 
     private readonly dependees: Map<ObjectField | ObjectNode, DisposingDependee> = new Map();
 
+    // All cursors that are in the "CUrrent" state. Must be em,pty when editing.
```

Comment:
Extraneous comma in "empty" and likely unintentionally capitalized "U" in "CUrrent"

Reply:
```suggestion
    // All cursors that are in the "Current" state. Must be empty when editing.
```